dspace-server-webapp/src/main/java/org/dspace/app/rest/authorization/impl/CanDownloadPDFAttestationFeature.java

```diff
@@ -0,0 +1,63 @@
+package org.dspace.app.rest.authorization.impl;
+
+import java.sql.SQLException;
+
+import org.dspace.app.rest.authorization.AuthorizationFeature;
+import org.dspace.app.rest.authorization.AuthorizationFeatureDocumentation;
+import org.dspace.app.rest.model.BaseObjectRest;
+import org.dspace.app.rest.model.ItemRest;
+import org.dspace.app.rest.utils.Utils;
+import org.dspace.authorize.service.AuthorizeService;
+import org.dspace.content.DSpaceObject;
+import org.dspace.content.Item;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.dspace.eperson.EPerson;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Rule to indicate who has the permission to download a given item's submission attestation
+ */
+@Component
+@AuthorizationFeatureDocumentation(name = CanDownloadPDFAttestationFeature.NAME,
+    description = "It can be used to verify if the attestation can be downloaded")
```

Comment:
COSMETIC & PERSONAL : more readable
```java
@AuthorizationFeatureDocumentation(
    name = CanDownloadPDFAttestationFeature.NAME,
    description = "It can be used to verify if the attestation can be downloaded"
)
```